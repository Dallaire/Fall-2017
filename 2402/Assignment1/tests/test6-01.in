qw
abc
a
efg
acd
ssqer
tttt
b
bc
