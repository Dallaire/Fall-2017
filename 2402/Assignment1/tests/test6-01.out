a
b
bc
qw
abc
acd
efg
tttt
ssqer
